libraries/AP_BattMonitor/AP_BattMonitor_config.h

```diff
@@ -119,3 +119,7 @@
 #ifndef AP_BATTERY_SCRIPTING_ENABLED
 #define AP_BATTERY_SCRIPTING_ENABLED (AP_SCRIPTING_ENABLED && AP_BATTERY_BACKEND_DEFAULT_ENABLED)
 #endif
+
+#ifndef AP_BATTERY_OPTIONS_PARAM_ENABLED
+#define AP_BATTERY_OPTIONS_PARAM_ENABLED (!defined(HAL_BUILD_AP_PERIPH) || AP_BATTERY_SUM_ENABLED)
+#endif
```

Comment:
Just need a new line at the end of the file to get rid of the stop sign thing.
```suggestion
#endif

```